hdfs3/core.py

```diff
@@ -413,6 +421,12 @@ def tail(self, path, size=None):
             f.seek(length - size)
             return f.read(size)
 
+    def head(self, path, size=None):
```

Comment:
Presumably we can add the number in the signature now?

Regardless this function is broken right now if the user says `hdfs.head()`.  We should test for this.